src/boot/ual.js

```diff
@@ -149,11 +149,12 @@ export default boot(async ({ app, store }) => {
             loginHandler,
             signHandler,
         }),
-        new OreIdAuthenticator([chain], {
-            appId: process.env.OREID_APP_ID_NATIVE,
-            plugins: { popup: WebPopup() },
-        },
-        AuthProvider.Google),
+        // TODO: What should we do with this Authenticator?
+        // new OreIdAuthenticator([chain], {
+        //     appId: process.env.OREID_APP_ID_NATIVE,
+        //     plugins: { popup: WebPopup() },
+        // },
+        // AuthProvider.Google),
```

Comment:
What should I do with this Authenticator?
It's the ORE-ID (old Cloud Wallet)

Reply:
We don't want to disable ORE ID yet or block this UI update. Can restore this functionality?